packages/pricing/package.json

```diff
@@ -55,6 +55,7 @@
     "@mikro-orm/postgresql": "5.7.12",
     "awilix": "^8.0.0",
     "dotenv": "^16.1.4",
-    "knex": "2.4.2"
+    "knex": "2.4.2",
+    "lodash": "^4.17.21"
```

Comment:
nit: we could bring the implementation of the method used to the repo to reduce the size of the package. It doesn't need to be in this PR though.

Reply:
+1